docs/user-guide/clients/otlp.md

```diff
@@ -0,0 +1,26 @@
+# OpenTelemetry Protocol(OTLP)
+
+[OpenTelemetry](https://opentelemetry.io/) is a vendor-neutral open-source Observability framework for instrumenting, generating, collecting, and exporting telemetry data such as traces, metrics, logs. The OpenTelemetry Protocol (OTLP) defines the encoding, transport, and delivery mechanism of telemetry data between telemetry sources, intermediate processes such as collectors and telemetry backends.
+
+GreptimeDB can be used as a collector to receive OpenTelemetry Metrics via the OTLP protocol.
+
+## OTLP/HTTP
+
+To send OpenTelemetry Metrics to GreptimeDB through Opentelemetry exporter SDK libraries, use the following information:
+
+* URL: `http://localhost:4000/v1/otlp/v1/metrics`
+* Method: `POST`
+* Headers:
+  * `Content-Type`: `application/x-protobuf`
+  * `X-Greptime-DB-Name`: The database name
+  * `Authorization`: `Basic` authentication, please refer to [Authentication](./authentication.md) and [HTTP API](./http-api.md#authentication)
```

Comment:
1. Avoid duplicated information 
2. We don't write OTLP client from scratch so some information here is not need (method, content-type)

We'd better provide an example using one of OpenTelemetry SDKs to guide user how to set these properties.

Reply:
1. There are some minor differences between the content in the user guide and the cloud integration. For example, the links in the cloud integration require the full URL, such as `https://docs.greptime.com/user-guide/clients/authentication`. It is also better to include duplicated content, such as the Base64 encoded string, to help console users quickly understand the authentication process. 
Perhaps we can use the same content by removing the minor differences and discarding the customized content.

2. Please see the answer above

3. OpenTelemetry SDKs are currently unstable, I recommand providing links to demos to let users know how to write the code using a specific version of OpenTelemetry.